docs/data/data-grid/filtering/filtering.md

```diff
@@ -47,6 +47,39 @@ To disable `Add filter` or `Remove all` buttons, pass `disableAddFilterButton` o
 
 {{"demo": "DisableActionButtonsDataGridPro.js", "bg": "inline", "defaultCodeOpen": false}}
 
+## Header filtering [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan)
```

Comment:
I believe this sounds better, and other components also don't overuse the "ing" (we should probably consider dropping the "ing" from the other filtering titles too– on a different PR).

```suggestion
## Header filters [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan)
```

Reply:
Adding another two cents: Maybe relate it to the quick filter but on a column level?
```suggestion
## Quick column filters [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan)
```

Or simply `Column filter`, what do you think?

Reply:
I think "Quick column filters" might be confusing given that we also have a "Quick filtering" feature.
"Header filters" sounds good to me though.